app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt

```diff
@@ -308,7 +317,7 @@ class AddCycleway : OsmElementQuestType<CyclewayAnswer> {
                 !maxspeed
                 or maxspeed > 20
                 or (maxspeed ~ ".*mph" and maxspeed !~ "([1-9]|1[0-2]) mph")
```

Comment:
In the UK on a "proper" road it's rare to see < 20mph (which I guess is handled in the case above). But 15, and more often 10 or occasionally 5mph are common on e.g. large private sites/roads. I don't know if it's worth relaxing it to include the likely mph equivalent, rather than a direct conversion of kph to mph?

Reply:
That's the "definitely never ask for a cycleway" filter. In roads where there is a speed limit of 20 or less (whatever the reason for this), there is absolutely no sense for a cycleway because that is the speed cyclists go.

Reply:
And to shift this limit solely on the base of the unit used for speed limit doesn't make sense to me. Cyclists don't go faster in the UK than in the rest of the world.

Reply:
Actually they do :laughing: (I was just looking for cycling speeds):
https://www.cyclingweekly.com/fitness/training/13-ways-increase-average-cycling-speed-144937

UK Men average 15mph, which means the filter should exclude all roads <= 15mph. FWIW I did find some 12mph tagging (round two UK parks), but 15mph is used 200 times more.

Anyway, I was more meaning that directly converting from kmh to mph is likely to result in unusual/infrequently used speed limits, potentially cutting off the realistic/actual ones, see this table for some more examples (going the other way):
https://www.sharetheroadtozero.com/wp-content/uploads/2016/03/Highway-Code-Speed-Limit-Infographic.png

Reply:
British ride faster because the constant rain cools them down ;-)

As said, this is not about speed limit signs but about some (TBH randomly) chosen speed at which cyclists go.  Look at the females, 20 km/h seems quite reasonable as a rough average. Also, how were the statistics created? Is this only for cycling enthusiasts (on racing bikes) or the average on any bike?

The case you seem to make here is that roads signed with 15 mph will never have a cycleway because cyclists are never expected to go slower than that. Is this the case you want to make?

Reply:
> British ride faster because the constant rain cools them down ;-)

I assume the Dutch win as there are no hills.

> As said, this is not about speed limit signs but about some (TBH randomly) chosen speed at which cyclists go. Look at the females, 20 km/h seems quite reasonable as a rough average. Also, how were the statistics created? Is this only for cycling enthusiasts (on racing bikes) or the average on any bike?

Strava is a tracking app, so it will probably have more enthusiasts, but possibly on mountain bikes and stuff too. People probably aren't using it to track their daily commute.

> The case you seem to make here is that roads signed with 15 mph will never have a cycleway because cyclists are never expected to go slower than that. Is this the case you want to make?

I think more broadly that 15 mph is so slow it's the safety factor, you don't then need to separate them. As this article alludes to, <20 mph is very rare in the UK on a normal road, if you give them a cycle lane too, you're only really protected pedestrians just randomly crossing. I'd imagine the reality here is there isn't the space to add a separate cycleway, so the limit is the alternative option:
https://inews.co.uk/essentials/lifestyle/cars/car-news/uk-set-to-get-its-first-15mph-speed-limits-295471